AGENTS.md

```
@@ -0,0 +1,533 @@
+- **Git Status:** ✅ **COMMITTED** (required for action to work)
+- **Used by:** GitHub Actions runners
+
+Example `dist/index.js`:
+```javascript
+/******/ var __webpack_modules__ = ({
+// All dependencies are webpack-bundled here
+```
+
+#### Build Pipeline Flow
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)